metabrainz/templates/index/gdpr.html

```diff
@@ -77,27 +77,27 @@ <h2>{{ _('Right to be forgotten') }}</h2>
   <h2>{{ _('Restriction of processing') }}</h2>
 
   <p>
-    Summary: <i>To allow the user to control how their personally identifying data is being used.</i>.
+    Summary: <i>To allow the user to control how their personally identifying data is being used.</i>
   </p>
   <p>
-    With the exception of the ListenBrainz and AcousticBrainz projects we do not process personally identifying data. Any personally 
-    identifying data that our projects store is for the purpose of being able to contact you about your contributions to our database 
+    Except for the ListenBrainz and AcousticBrainz projects, we do not process personally identifying data. Any personally 
+    identifying data that our projects store is to be able to contact you about your contributions to our database 
```

Comment:
I feel like the "for the purpose of" might be GDPR language? So may we want to keep that? I agree with the change to the first line. (I also don’t object to the change in the 2nd line if it isn’t GDPR derived.)